scalableminds/webknossos
# Freeze volume versions

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- trace some in volume mode
- stop wK, change `tracingstore.freezeVolumeVersions` to `true`
- run wK again, open same volume, trace some more
- no new versions should be created

### Issues:
- possible contribution to #3528

------
